src/importmesh.cpp

```diff
@@ -183,35 +212,34 @@ bool LinkStl(const Platform::Path &filename, EntityList *el, SMesh *m, SShell *s
     }
 
     hEntity p[8];
-    int id = el->n+2;
```

Comment:
What was this for? I obviously have not gone deep enough to understand.

Reply:
I honestly don't recall. I may have wanted to leave a gap in there to add a normal? It turns out the 3 normals require 4 entities if they share the same point.